Source/WarpX.H

```diff
@@ -778,6 +778,10 @@ public:
 
     void SyncRho ();
 
```

Comment:
I would expect that, similarly to what was done in #3277, it should be possible to remove this function and update the Python interface accordingly. I guess this would require to modify both Source/Python/WarpXWrappers.H and Source/Python/WarpXWrappers.cpp. Maybe you initially modified only the cpp file and that's why it seemed like that change needed to be reverted in order to fix the Python interface?

Reply:
Yes, I decided not to change the Python interface after realizing that it is being called here:
https://github.com/ECP-WarpX/WarpX/blob/development/Python/pywarpx/_libwarpx.py#L1219
Changing this would require adding a Python interface to extract the list of `rho_fp` and `rho_cp` at each level, so that the Python code would instead read:
```
self.libwarpx_so.warpx_SyncRho( self.libwarpx_so.warpx_GetRhoFP(), self.libwarpx_so.warpx_GetRhoCP() )
```
While there are existing Python interfaces that return the `rho_fp` and `rho_cp` for **one given level**, we don't have one yet for the full list of levels. I think that this could be a follow-up PR, potentially after the transition to `pyamrex`.
Any opinion on your side?

Reply:
Incidentally, note that we have a call to `warpx_SyncCurrent()` here:
https://github.com/ECP-WarpX/WarpX/blob/development/Python/pywarpx/timestepper.py#L60
which in principle should fail because it does not conform to the syntax defined in #3277 - but it does not seem that this file is being tested. Maybe we could discuss this offline.

Reply:
I would suggest keeping `SyncRho()` for now and deal with this after transitioning to pyamrex.
Also, for now I wouldn't worry about timestepper.py. It has probably never been really tested and it likely to have other problems. This is also something that can be fixed after pyamrex.